setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
+    keywords=["OCR", "OpenCV"],
+    license="AGPL",
+    classifiers=["Programming Language :: Python :: 2.7",
+                 "Programming Language :: Python :: 3",
+                 "License :: OSI Approved :: GNU Affero General Public License v2 or later (AGPLv2+)"],
+    include_package_data=True,
+    install_requires=["opencv-python"]
```

Comment:
This `install_requires` statement cannot be handled by doing a `python setup.py install`, so the package must be installed with `python -m pip install .`, so `pip` can check if `opencv-python` is installed, and install it if necessary.